bridges/snowbridge/primitives/core/src/location.rs

```diff
@@ -230,4 +249,132 @@ mod tests {
 			);
 		}
 	}
+
+	#[test]
+	fn test_agent_id_fully_compatible() {
+		// Agent locations in V1 are not reanchored
```

Comment:
I think its important to add in the docs for `LocationHashOf` that the legacy case is only triggered when v1 code passes an agent location that is not reanchored.

Currently its hard to understand what `LocationHashOf` does, and I had to read the tests to find out.

Reply:
LocationHashOf is just for V2

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/28/commits/dcb8d7b419a0294b3c9a4e7cc5b84c2fb97595a3